.scalafmt.conf

```diff
@@ -1,9 +1,10 @@
-version = 2.7.5
+version = 3.0.4
```

Comment:
Maybe have a look at what they are doing upstream, the goal being to not diverge too much (and formatting would be a bad reason to increase the diff).

https://github.com/akka/akka-persistence-jdbc/commits/master/.scalafmt.conf

Reply:
I've synced the scalafmt settings with the ones present on `akka-persistence-jdbc`. Also updated to 3.0.5, since no changes are introduced by it.